### PR TITLE
Issue #SC-1418 added itext license for supporting glyphs in pdf gener…

### PR DESCRIPTION
**Pull request by Hari-stackroute**
…ation

### CODE REVIEW

File: certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfConverter.java [hunk 1]
```diff
@@ -13,6 +19,12 @@
     public static void convertor(File htmlSource, String certUuid, String directory) {
         File file = new File(directory, certUuid + ".pdf");
         try {
+            boolean licenseEnabled = Boolean.parseBoolean(System.getenv(JsonKey.ITEXT_LICENSE_ENABLED));
+            if(licenseEnabled) {
```
Comment (manzarul) [inline certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfConverter.java]: insdie if we can directly pass : Boolean.parseBoolean(System.getenv(JsonKey.ITEXT_LICENSE_ENABLED))
* i think we need to handle both scenario (if license is enabled and path is provided then only load it ), in other case use default  and add one logger that will let you know what user has provided.

Comment (Hari-stackroute) [reply certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfConverter.java]: @manzarul even though if license is loaded or not remaining works same. added logger.
Done accordingly.

File: certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfConverter.java [hunk 2]
```diff
@@ -13,6 +19,12 @@
     public static void convertor(File htmlSource, String certUuid, String directory) {
         File file = new File(directory, certUuid + ".pdf");
         try {
+            if(Boolean.parseBoolean(System.getenv(JsonKey.ITEXT_LICENSE_ENABLED))) {
```
Comment (manzarul) [inline certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfConverter.java]: if (Boolean.parseBoolean(System.getenv(JsonKey.ITEXT_LICENSE_ENABLED)) && StringUtils.notEmpty(System.getenv(JsonKey.ITEXT_LICENSE_PATH)))

Comment (Hari-stackroute) [reply certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfConverter.java]: ok done.

**Pull request merged**